### ISSUE TITLE
Add a crash reporting tool?

### ISSUE THREAD

**Issue by kkimdev** [m19g/www.m19g.com#36]
sentry.io?

**Comment by suj97** [comment]
Will be useful for the online-test platform and web-forum. I don't think it's needed for a static website.

**Comment by kkimdev** [comment]
Yes it will be less useful, but because we're using Angular, still there is a non-trivial amount of javascript being executed.  Actually I was purposefully overtooling, including Angular, Travis, etc, for a simple static company website as a testing/learning ground for other more dynamic websites that we will be building.
